requirements.txt

```diff
@@ -2,4 +2,5 @@ Mako
 PyYAML
 six
 pbr
-defusedxml
+# For OpenNebula provider
+# defusedxml
```

Comment:
Yes it seems definitely appropriate, even if it will probably be mostly used by devs as site admins will (I hope...) use packages.
I already created two releases today (aheum...) but we can include this in the next one.